src/components/atoms/FileButton/FileButton.tsx

```diff
@@ -1,57 +1,68 @@
-import React, { useCallback } from "react";
+import React, { useCallback, useRef } from "react";
 import classNames from "classnames";
 
+import { ButtonNG, ButtonProps } from "../ButtonNG/ButtonNG";
+
 import "./FileButton.scss";
 
-export interface FileButtonProps {
-  title: string;
-  description?: string;
-  disabled?: boolean;
-  onChange: (url: string, file: FileList) => void;
+export interface FileButtonOnChangeData {
+  url: string;
+  files: FileList;
+  file: File;
+}
+
+export interface FileButtonProps extends Omit<ButtonProps, "onClick"> {
+  onClick: (data: FileButtonOnChangeData) => void;
 }
 
 export const FileButton: React.FC<FileButtonProps> = ({
-  title,
-  description,
-  disabled: isDisabled,
-  onChange,
+  disabled,
+  onClick,
+  className,
+  children,
+  ...buttonNgProps
 }) => {
-  const handleChange = useCallback(
-    (event: React.ChangeEvent<HTMLInputElement>) => {
-      const files = event.target.files;
+  const formRef = useRef<HTMLFormElement>(null);
+  const inputId = `FileButton-input-id-${new Date().getTime()}-${Math.random()}`;
 
-      if (!files || isDisabled) return;
+  const onChange = useCallback(
```

Comment:
This is a handler and shouldn't be named with `on` prefix. Only callback props are named with it.